package.json

```diff
@@ -12,14 +12,13 @@
     "clean": "node scripts/clean.js",
     "build:ts": "tsc",
     "build:gitignore": "node scripts/update_gitignore_for_tsc_output.js run",
-    "format": "prettier --write \"{lib,bin,scripts,test}/**/*.js\"",
-    "format:check-difference": "prettier --list-different \"{lib,bin,scripts,test}/**/*.js\"",
+    "format": "prettier $([ ! $CI ] && echo '--write') --list-different .",
     "debug:test": "mocha --inspect-brk --exit -t 0 test/all-tests-suite.js",
     "debug:tape": "node --inspect-brk test/tape/index.js",
     "coveralls": "nyc report --reporter=lcov",
-    "lint": "eslint \"lib/**/*.js\" \"test/**/*.js\" \"bin/**/*.js\"",
+    "lint": "eslint --cache $([ ! $CI ] && echo '--fix') '**/*.js'",
     "lint:types": "tsd && dtslint types",
-    "lint:everything": "npm run lint:types && npm run lint",
+    "lint:everything": "npm run format && npm run lint && npm run lint:types",
```

Comment:
linting and formatting are different things, they should not be combined

Reply:
This is true but since prettier does a little linter I figured it wouldn't be a problem to combine. Do you prefer that I need it separate?

Reply:
yes, otherwise there is a blind spot if CI preformats something and then says oh yeah, it all passes linting

Reply:
I'm happy to separate it but just to the clear the setup in this PR doesn't have that blind spot. It runs prettier in check mode so it can fail PRs with things like files using the wrong line breaks (like it happened in #5455).

Reply:
Done.